### ISSUE TITLE
[HOLD] The "Delete Comment" context tab has been moved to the left

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#16639]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to URL https://staging.new.expensify.com/
2. Login with any account
3.Open any chat
4. Click on the Trash icon to delete the message

## Expected Result:
The "Delete Comment" tab should be in the center

## Actual Result:
The Delete Comment tab has been moved to the left.
Deleting Ws the same issue and Ctrl+I

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** 1.2.91-0
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
![Bug5996210_delete_WS_28 03](https://user-images.githubusercontent.com/43996225/228331034-742e1d45-b148-4967-8700-08d4c35dc225.png)

https://user-images.githubusercontent.com/43996225/228331038-a9811739-0019-48d2-b506-0f410c4bb1bc.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause internal team
**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @strepanier03 (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by MelvinBot** [comment]
Auto-assign attempt failed, all eligible assignees are OOO.

**Comment by varshamb** [comment]
I have reported this bug on slack.

https://expensify.slack.com/archives/C049HHMV9SM/p1680023847655009

**Comment by situchan** [comment]
This is duplicated with #16636 since they have the same root cause

**Comment by strepanier03** [comment]
Putting this one on HOLD until the linked GH is resolved. Then I'll test and make sure.

**Comment by luacmartins** [comment]
We reverted the offending PR. This should be fixed once the staging CP is finished.

**Comment by luacmartins** [comment]
We shipped the fix and this is solved. Closing.

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.92-2 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/16659


If no regressions arise, payment will be issued on 2023-04-07. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**
